kotlinx.fuzz.jazzer/src/main/kotlin/kotlinx/fuzz/jazzer/JazzerLauncher.kt

```diff
@@ -53,8 +51,10 @@ object JazzerLauncher {
         exitProcess(0)
     }
 
-    @OptIn(ExperimentalPathApi::class)
-    fun runTarget(instance: Any, method: Method): Throwable? {
+    private fun countCrashes(reproducerPath: Path) =
+        reproducerPath.listDirectoryEntries("{crash-*,timeout-*,slow-imput-*}").size.toLong()
```

Comment:
Typo: "i**m**put"

Reply:
Also, `reproducerPath.listDirectoryEntries("{crash-*,timeout-*,slow-imput-*}")` as separete function will be usefull futher

Reply:
Fixed